### PR TITLE
scintilla: fix 32 bit build

### PR DESCRIPTION
**Pull request by jackburton79**
Fix scintilla 32bit build (#8645). Hopefully next version of Scintilla will fix this upstream

### CODE REVIEW

**Comment by jackburton79** [comment]
Done bumping revision

File: x11-libs/scintilla/scintilla-5.3.4.recipe [hunk 1]
```diff
@@ -13,13 +13,13 @@ colours and multiple fonts."
...
 LICENSE="Scintilla"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://prdownloads.sf.net/scintilla/scintilla${portVersion//\./}.tgz"
 CHECKSUM_SHA256="3f01b1aef2b7e98f628af2cff965876f5d15ee2801d9df96dd3aced8f087cb46"
 SOURCE_DIR="scintilla"
 SOURCE_URI_2="https://prdownloads.sf.net/scintilla/scintillahaiku${portVersion//\./}.tgz"
 CHECKSUM_SHA256_2="9a4d46ebca064bc4484f1bc33e765774b2af669d61d4a26ddd4aa8cda1536c27"
-
+PATCHES="scintilla-$portVersion.patchset"
```
Comment (Begasus) [inline x11-libs/scintilla/scintilla-5.3.4.recipe:22]: Could you add a empty line between PATCHES and ARCHITECTURES? (guidelines)

**Comment by jackburton79** [comment]
Done

**Comment by Begasus** [comment]
Thanks! Has this been reported upstream?

**Pull request merged**